Framework/Core/src/DeviceSpecHelpers.cxx

```diff
@@ -1664,7 +1664,10 @@ boost::program_options::options_description DeviceSpecHelpers::getForwardedDevic
     ("infologger-mode", bpo::value<std::string>(), "O2_INFOLOGGER_MODE override")                                                                                    //
     ("infologger-severity", bpo::value<std::string>(), "minimun FairLogger severity which goes to info logger")                                                      //
     ("dpl-tracing-flags", bpo::value<std::string>(), "pipe separated list of events to trace")                                                                       //
-    ("child-driver", bpo::value<std::string>(), "external driver to start childs with (e.g. valgrind)");                                                             //
+    ("child-driver", bpo::value<std::string>(), "external driver to start childs with (e.g. valgrind)")                                                              //
+    ("inspector-address", bpo::value<std::string>()->default_value("127.0.0.1"), "address of DataInspector proxy")                                                   //
+    ("inspector-port", bpo::value<std::string>()->default_value("8081"), "port of DataInspector proxy")                                                              //
+    ("inspector-id", bpo::value<std::string>()->default_value(""), "id of analysis started by DataInspector proxy");
```

Comment:
I would prefer if you kept these options as environment variables, so that you do not affect the framework code with some plugin detail. At some point I will provide a way for the plugins to add options, but for now, I think a "getenv" would be better.

Reply:
We were looking for other ways to add options, but we hadn't thought abut environment variables. It is much better idea, so we will try to fix it ASAP.

Reply:
Thanks. After that is done, I think I will have no further comments.